public/docs/ts/latest/guide/architecture.jade

```diff
@@ -340,19 +326,18 @@ figure
   as with event binding.
 
   Angular processes *all* data bindings once per JavaScript event cycle,
-  from the root of the application component tree down to the leaves.
+  from the root of the application component tree through all components.
```

Comment:
Direction is important here. Maybe:

from the root of the application component tree down through all the components it contains, directly or indirectly.

(Wow, much longer without being much clearer. Maybe you can improve it?)